fastlane/Fastfile

```diff
@@ -1054,6 +1054,19 @@ inal copy, and try again.")
     UI.message "Checked out branch `#{git_branch}` as requested by user.\n"
   end
 
+  def commit_version_bump
+    version_file = PUBLIC_CONFIG_FILE
+    git_add(
+      path: version_file,
+      shell_escape: false
+    )
+    git_commit(
+      path: version_file,
+      message: 'Bump version number',
+      allow_nothing_to_commit: false
+    )
+  end
```

Comment:
[In WordPress-iOS](https://github.com/wordpress-mobile/WordPress-iOS/blob/dfa631f130ee22498aaf920ddb49ace9dbe6811b/fastlane/lanes/release.rb#L467-L473) I too started with `git_add` and `git_commit` but then [removed it](https://github.com/wordpress-mobile/WordPress-iOS/commit/69a7414f2148c6ca96e5f82d06c4bd2b2fb565b1) because Fastlane's `git_commit` also adds the files.

The reason I thought to use `git_add` was because my `git_commit` call failed the first time. But that turned out to be due to case sensitivity in the files path.

Reply:
Shall we remove `git_add` then?

Reply:
Good to know! Removed in https://github.com/Automattic/pocket-casts-ios/commit/68bcf786a4a954a4325dff56064236faf1ed7a11

Reply:
Calling `git_add` before `git_commit` is only necessary for brand new files that were not under version control until now. If the file was already tracked in git and all you wanted to do is stage the changes of the existing file, just calling `git_commit` is enough.

iinm, this behavior is consistent with `git add` and `git commit -a` (or `git commit {list of files}`) in `git` itself, not really specific to those _fastlane_ actions. For example `git commit -a` will add all the already-tracked-but-modified files to the commit you're creating, but won't add any new file that wasn't tracked in git yet, and you'll have to call `git add` explicitly on those new files first if you want to start tracking them and include them in your commit.